test/unit/org/apache/cassandra/db/commitlog/CommitLogTest.java

```diff
@@ -476,69 +476,6 @@ public void testEqualRecordLimit() throws Exception
         CommitLog.instance.add(rm);
     }
 
-    @Test(expected = MutationExceededMaxSizeException.class)
-    public void testExceedRecordLimit() throws Exception
-    {
-        Keyspace ks = Keyspace.open(KEYSPACE1);
-        ColumnFamilyStore cfs = ks.getColumnFamilyStore(STANDARD1);
-        Mutation rm = new RowUpdateBuilder(cfs.metadata(), 0, "k")
-                      .clustering("bytes")
-                      .add("val", ByteBuffer.allocate(1 + getMaxRecordDataSize()))
-                      .build();
-        long cnt = CommitLog.instance.metrics.oversizedMutations.getCount();
-        try
-        {
-            CommitLog.instance.add(rm);
-        }
-        catch (MutationExceededMaxSizeException e)
-        {
-            assertEquals(cnt + 1, CommitLog.instance.metrics.oversizedMutations.getCount());
-            throw e;
-        }
-        throw new AssertionError("mutation larger than limit was accepted");
-    }
-    @Test
-    public void testExceedRecordLimitWithMultiplePartitions() throws Exception
-    {
-        CommitLog.instance.resetUnsafe(true);
-        List<Mutation> mutations = new ArrayList<>();
-        Keyspace ks = Keyspace.open(KEYSPACE1);
-        char[] keyChars = new char[MutationExceededMaxSizeException.PARTITION_MESSAGE_LIMIT];
-        Arrays.fill(keyChars, 'k');
-        String key = new String(keyChars);
-
-        // large mutation
-        mutations.add(new RowUpdateBuilder(ks.getColumnFamilyStore(STANDARD1).metadata(), 0, key)
-                      .clustering("bytes")
-                      .add("val", ByteBuffer.allocate(1 + getMaxRecordDataSize()))
-                      .build());
-
-        // smaller mutation
-        mutations.add(new RowUpdateBuilder(ks.getColumnFamilyStore(STANDARD2).metadata(), 0, key)
-                      .clustering("bytes")
-                      .add("val", ByteBuffer.allocate(1 + getMaxRecordDataSize() - 1024))
-                      .build());
-
-        Mutation mutation = Mutation.merge(mutations);
-        try
-        {
-            CommitLog.instance.add(Mutation.merge(mutations));
-            throw new AssertionError("mutation larger than limit was accepted");
-        }
-        catch (MutationExceededMaxSizeException exception)
-        {
-            String message = exception.getMessage();
-
-            long mutationSize = mutation.serializedSize(MessagingService.current_version) + ENTRY_OVERHEAD_SIZE;
```

Comment:
Nit: this leaves the static import of `ENTRY_OVERHEAD_SIZE` usued.

Reply:
Done in [5d7f657](https://github.com/apache/cassandra/pull/1588/commits/5d7f657cd1a3c8a5d841e99d1a3d670c0efddd53).